lib/edit-foods.js

```diff
@@ -1,14 +1,23 @@
 $(document).ready(function() {
   $('body').on('click', '.food-row td', function() { 
-    var $el = $(this);
-    var $input = $('<input/>').val( $el.text() );
-    $el.replaceWith( $input );
+    var $td = $(this);
+    var food = localStorage.getItem( $td.text() );
+    var $input = $('<input/>').val( $td.text() );
+    $td.replaceWith( $input );
     
-    var save = function() {  
-      var $td = $('<td />').text($input.val() );
-      $input.replaceWith($td);
+    var save = function() {
+      localStorage.setItem($input.val(), `<tr class='food-row'>
+                <td class='food-name'>` + $input.val() + `</td>
+                <td class='food-calories'>` + $input.siblings().first().text()  + `</td>
+                <td class='food-delete'><button>-</button></td>
+              </tr>`); 
```

Comment:
I agree that would be a lot more clean. Soon, we won't be passing in this whole mess, thankfully. Once we have localStorage update, I'll move the new logic into a variable.